internal/cluster/transition_test.go

```diff
@@ -3728,6 +3728,36 @@ var _ = Describe("Ocs Operator use-cases", func() {
 			}),
 			errorExpected: false,
 		},
+		{
+			name:               "ocs enabled, 3 nodes, one with empty inventory",
+			srcState:           models.ClusterStatusReady,
+			dstState:           models.ClusterStatusInsufficient,
+			machineNetworkCidr: "1.2.3.0/24",
+			apiVip:             "1.2.3.5",
+			ingressVip:         "1.2.3.6",
+			dnsDomain:          "test.com",
+			pullSecretSet:      true,
+			hosts: []models.Host{
+				{ID: &hid1, Status: swag.String(models.HostStatusKnown), Inventory: ocsInventoryWithDisks(12, 64000000000), Role: models.HostRoleMaster},
+				{ID: &hid2, Status: swag.String(models.HostStatusKnown), Inventory: ocsInventoryWithDisks(12, 64000000000), Role: models.HostRoleMaster},
+				{ID: &hid3, Status: swag.String(models.HostStatusKnown), Inventory: "", Role: models.HostRoleMaster},
+			},
```

Comment:
@gamli75  
I have added a test case where the host inventory will be empty, which means that after disable when host is enabled again, the inventory will be empty. Let me know if some other test-case can be added.